src/Backend/Modules/Blog/Domain/Comment.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace Backend\Modules\Blog\Domain;
+
+use Common\Locale;
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="blog_comments")
+ * @ORM\Entity(repositoryClass="Backend\Modules\Blog\Domain\CommentRepository")
+ * @ORM\HasLifecycleCallbacks
+ */
+class Comment
+{
+    /**
+     * @var int
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @var int
+     * @ORM\Column(type="integer", name="post_id")
+     */
+    private $postId;
+
+    /**
+     * @var Locale
+     * @ORM\Column(type="locale", name="language")
+     */
+    private $locale;
+
+    /**
+     * @var string
+     * @ORM\Column(type="string")
+     */
+    private $author;
+
+    /**
+     * @var string
+     * @ORM\Column(type="string")
+     */
+    private $email;
+
+    /**
+     * @var string
+     * @ORM\Column(type="string", nullable=true)
+     */
+    private $website;
+
+    /**
+     * @var string
+     * @ORM\Column(type="text")
+     */
+    private $text;
+
+    /**
+     * @var string
+     * @ORM\Column(type="string", options={"default":"comment"})
+     */
+    private $type;
+
+    /**
+     * @var string
+     * @ORM\Column(type="string", options={"default":"moderation"})
+     */
+    private $status;
+
+    /**
+     * @var string
+     * @ORM\Column(type="text", nullable=true)
+     */
+    private $data;
+
+    /**
+     * @var DateTime
+     * @ORM\Column(type="datetime", name="created_on")
+     */
+    private $createdOn;
+
+    public function __construct(
+        int $postId,
+        Locale $locale,
+        string $author,
+        string $email,
+        string $website,
```

Comment:
In the mapping you state `website` is nullable, yet here you require it to be a string